common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -99,3 +99,18 @@
 else:
     tmpl.render_context(context)
 %></%def>
+
+<%def name="optional_include_mako_with_fallback(file, fallback, with_microsite=False, **kwargs)"><%
```

Comment:
AS for me there are two possible variants: 1. shouldn't handle multiple fallbacks at all (template author should provide provide the logic into his template), 2. to consider fallback argument as a list of templates.
The variant with two separate arguments (one for the first fallback and one for the others) looks strange. Also we could consider fallback param as one fallback if it was passed as s string and as a group of fallbacks if it was passed as a list. But I think this variant strange and non-intuitive as well.